modules/bim/spec/features/bcf/bcf_snapshot_column_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe 'BCF snapshot column', type: :feature, js: true, with_mail: false do
+  let(:project) { FactoryBot.create(:project) }
+  let(:wp_table) { Pages::WorkPackagesTable.new(project) }
+  let(:permissions) { %i[add_work_packages view_work_packages view_linked_issues] }
+  let!(:work_package) { FactoryBot.create(:work_package, project: project) }
+  let!(:bcf_issue) { FactoryBot.create(:bcf_issue_with_viewpoint, work_package: work_package) }
+  let(:user) do
+    FactoryBot.create :user,
+                      member_in_project: project,
+                      member_with_permissions: permissions
+  end
+
+
+  before do
+    login_as(user)
+  end
+
+  let!(:query) do
```

Comment:
Just for code quality issues: Please move this definition to other ones above the `before` block. :)